assets/src/stories-editor/blocks/amp-story-cta/save.js

```diff
@@ -27,16 +29,20 @@ const CallToActionSave = ( { attributes } ) => {
 
 	styles.top = btnPositionTop ? `${ btnPositionTop }%` : undefined;
 	styles.left = btnPositionLeft ? `${ btnPositionLeft }%` : undefined;
+	styles.width = btnWidth ? `${ btnWidth }%` : undefined;
+	styles.height = btnHeight ? `${ btnHeight }%` : undefined;
 
 	return (
 		<amp-story-cta-layer id={ anchor ? anchor : getUniqueId() }>
-			<RichText.Content
-				tagName="a"
-				className={ className }
-				href={ url }
-				style={ styles }
-				value={ text }
-			/>
+			<div className="amp-cta-button-wrapper">
+				<RichText.Content
```

Comment:
I'm wondering if it would make sense to use `amp-fit-text` here instead, for responsiveness. Thoughts?

Reply:
Makes sense to me.

Just using that by default could work. Can't think of a scenario where it makes sense being able to disable `amp-fit-text`.